clients/drcachesim/common/options.cpp

```diff
@@ -177,6 +177,12 @@ droption_t<bytesize_t> op_L0D_size(
     "Must be a power of 2 and a multiple of -line_size, unless it is set to 0, "
     "which disables data entries from appearing in the trace.");
 
+droption_t<bool> op_inst_only_trace(
```

Comment:
Given that existing options involving instructions use `instr` to abbreviate it (`op_trace_after_instrs`, `op_online_instr_types`) I would vote for consistency here: so `op_instr_only_trace`.

Reply:
Done.